app/Models/AlertSchedule.php

```diff
@@ -54,8 +54,8 @@ public function scopeIsActive($query)
                             ->where(function ($query) {
                                 $query->where('end_recurring_dt', '>=', DB::raw("date_format(NOW(), '%Y-%m-%d')"))
                                     ->orWhereNull('end_recurring_dt')
-                                    ->orWhere('end_recurring_dt', '0000-00-00')
-                                    ->orWhere('end_recurring_dt', '');
+                                    ->orWhere('end_recurring_dt', '0000-00-00');
+                                    //->orWhere('end_recurring_dt', '');
```

Comment:
instead of commenting out, remove it

Reply:
> instead of commenting out, remove it

Sure, NP - done! Was trying to be careful, but I see your point.

Thanks!

Reply:
therefore versioning system exists ... ;-) it's still in history

Reply:
OK, you lost me there, sorry. Not quite sure what you mean - can you clarify?

Thanks!

Reply:
it's a commit what you've done. if value is missed in commit history changes can be seen.
For Example: https://github.com/librenms/librenms/commits/master/includes/functions.php
here you can see every change is done ... so recovery is everytime possible

Reply:
Right, so I have both commits here - the comment, then the removal. Are you not wanting both? I get the feeling this is a git / commit item, just not quite sure how you want it set up.

Thanks!

Reply:
nope, everything fine.
Here in Pull Request everyone can see what's changed. This can be seen on every commit you take a look on. Independent how old or big a commit is.

Reply:
OK, perfect! Do you need me to do anything else then, or is this good to be merged? Just wasn't sure if I needed to do anything ... LOL!

Reply:
i think everything is fine. Will test it later

Reply:
Sounds good - thanks for following up. Have a great day!